transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/ButtonResponse.kt

```diff
@@ -9,9 +9,32 @@ import kotlinx.serialization.Serializable
  * @param payload the payload of the quick reply.
  * @param type the type of quick reply.
  */
+
 @Serializable
 data class ButtonResponse(
     val text: String,
     val payload: String,
-    val type: String,
-)
+) : Action(type = Type.Button)
+
+@Serializable
+data class LinkAction(
```

Comment:
We have to provide documentation to any public facing object/api. 
Please add few words about purpose of this class and/or its parameters. (same goes for PostbackAction)